auacm/app/modules/submission_manager/views.py

```diff
@@ -69,3 +69,44 @@ def submit():
     return serve_response({
         'submissionId': attempt.job
     })
+
+
+@app.route('/api/submit/job/<int:job_id>')
+def get_submit(job_id):
+    """Get the current status of a submission based on its id"""
+    submit = (session.query(models.Submission)
+              .filter(models.Submission.job == job_id).first())
+
+    if not submit:
+        return serve_error('Submission with job id ' + str(job_id) +
+                           ' not found', 401)
+
+    return serve_response(submit.to_dict())
+
+
+@app.route('/api/submit/user/<username>')
+@app.route('/api/submit/user/<username>/<int:limit>')
```

Comment:
* Yeah, I kind of hate the `../me` thing. Let's just make the client specify a username.
* I'm fine with leaving it outside the users route. It's not really a property of users, and the query string makes the most sense here.
* We can leave the `submit` route as-is for now. You're right that it's not too related to this pull, but since this pull is making the submissions manager more fully fleshed-out, it kind of makes sense. Also, I still lean towards `submissions` not `submission`, because you are creating something that will be a part of the submissions collection. Wikipedia talks about it more [here](https://en.wikipedia.org/wiki/Representational_state_transfer#Example).

So, can you change the route to `/api/submissions` for this one and `/api/submissions/<int:job>` for the other?